easybuild/easyconfigs/p/Python/Python-2.7.17-GCCcore-9.3.0.eb

```diff
@@ -0,0 +1,293 @@
+name = 'Python'
+version = '2.7.17'
+
+homepage = 'https://python.org/'
+description = """Python is a programming language that lets you work more quickly and integrate your systems
+ more effectively."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['f22059d09cdf9625e0a7284d24a13062044f5bf59d93a7f3382190dfa94cecde']
+
+dependencies = [
+    ('binutils', '2.34'),  # required for pip install that involves compilation
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.0'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.31.1'),
+    ('GMP', '6.2.0'),  # required for pycrypto
```

Comment:
Compared to Python 3.8.2 XZ is missing here as a dependency, why?

Reply:
XZ is a dependency in almost all of the Python 3 easyconfigs but is not a dependency in any Python 2 easyconfig.

Reply:
XZ is a new dep for Python 3, it's not required for Python 2